cypress/integration/e2e/commercial.spec.js

```diff
@@ -0,0 +1,27 @@
+import { getPolyfill } from '../../lib/polyfill';
+import { fetchPolyfill } from '../../lib/config';
+
+describe('Commercial E2E tests', function () {
+    before(getPolyfill);
+
+    beforeEach(function () {
+        // Disable CMP
+        cy.setCookie('gu-cmp-disabled', 'true', {
+            log: true,
+        });
+
+    });
+
+    describe('for WEB', function () {
+        // eslint-disable-next-line mocha/no-setup-in-describe
+        it(`It should check slots for a long article`, function () {
+                const longReadURL = "https://www.theguardian.com/environment/2020/oct/13/maverick-rewilders-endangered-species-extinction-conservation-uk-wildlife";
+                cy.log(`Opening A long read`);
+                cy.visit(`Article?url=${longReadURL}`, fetchPolyfill);
+
+                cy.scrollTo('bottom', { duration: 4500 });
```

Comment:
Hopefully this duration won't need to be extended but I've seen it in other places (I think the BBC also had similar issues) where this was flakey. But I don't see a better way as things stand so 👍 

Note. Hopefully, in some bright future, we will have callbacks like `onAdLoad` that we can use to count things like this 🤞

Reply:
I've switched now to 500ms after I realised the long duration was only needed while testing with --headed flag 🙂